src/rgw/rgw_sal_store.h

```diff
@@ -71,7 +71,6 @@ class StoreUser : public User {
 
 class StoreBucket : public Bucket {
   protected:
-    RGWBucketEnt ent;
```

Comment:
I'm trying to do a QA run, and I'm getting compiler errors because `ent` was removed and there are still references to it.

```
/home/jenkins-build/build/workspace/ceph-dev-new-build/ARCH/x86_64/AVAILABLE_ARCH/x86_64/AVAILABLE_DIST/centos9/DIST/centos9/MACHINE_SIZE/gigantic/release/18.0.0-4522-gc1b6c062/rpm/el9/BUILD/ceph-18.0.0-4522-gc1b6c062/src/rgw/rgw_sal_store.h:104:7: error: ‘ent’ was not declared in this scope; did you mean ‘int’?
  104 |       ent.count = _count;
      |       ^~~
      |       int
```